test/vstest.console.UnitTests/Internal/ConsoleLoggerTests.cs

```diff
@@ -1022,13 +1103,17 @@ public void GetTestMessagesShouldWriteMessageAndStackTraceToConsole()
             this.mockOutput.Verify(o => o.WriteLine(" AdditionalInfoCategory AnotherAdditionalInfoCategory", OutputLevel.Information), Times.Once());
         }
 
+        [DataRow("quiet")]
+        [DataRow("Normal")]
```

Comment:
Why is normal special ? with a capital letter I mean

Reply:
changed